aspnetcore/fundamentals/dependency-injection/samples/3.x/DependencyInjectionSample/Models/Operation.cs

```diff
@@ -4,21 +4,32 @@
 namespace DependencyInjectionSample.Models
 {
     #region snippet1
-    public class Operation : IOperationTransient, 
-        IOperationScoped, 
-        IOperationSingleton, 
+    public class Operation : IOperationTransient,
+        IOperationScoped,
+        IOperationSingleton,
         IOperationSingletonInstance
     {
-        public Operation() : this(Guid.NewGuid())
+        public Operation() : this(DateTime.Now.ToString("hh:mm:ss.fff"))
         {
+            DoWork();
         }
 
-        public Operation(Guid id)
+        public Operation(string id)
```

Comment:
@dougbu  here is the change from GUID to string. This makes it easy to see what has changed and what stays the same.